### ISSUE TITLE
CI integration test

### ISSUE THREAD

**Issue by bentsherman** [nextflow-io/nf-hello#14]
We should add a basic integration test to the CI that simply runs a nextflow pipeline with the plugin enabled.

For nf-hello it would probably just be a local pipeline script showing the customer channel factory, operator, etc, but different plugins will have different use cases. The important thing is to create an example here and then encourage plugin developers to do the same for their plugins (and we would do the same for nf-prov and nf-sqldb).

This will give people better confidence that plugin releases actually work with some version of Nextflow.

**Comment by pditommaso** [comment]
Done https://github.com/nextflow-io/nf-hello/blob/master/.github/workflows/build.yml

**Comment by bentsherman** [comment]
Not quite. The goal of this issue is to include a simple pipeline execution to test the plugin, but the current CI only builds the plugin
